sonar-scanner-cli.yaml

```
@@ -0,0 +1,44 @@
+package:
+  name: sonar-scanner-cli
+  version: 5.0.1.3006
+  epoch: 0
+  description: Scanner CLI for SonarQube and SonarCloud
+  copyright:
+    - license: GNU Lesser General Public License v3.0
```

Needs to be valid short [SPDX Identifier ](https://spdx.org/licenses/LGPL-3.0-or-later.html)
```suggestion
    - license: LGPL-3.0-or-later
```




```